assets/css/navigation.css

```diff
@@ -39,7 +39,7 @@ header {
 
     #right-navigation-block {
         justify-content: end;
-        flex-wrap: wrap;
+        flex-wrap: nowrap;
```

Comment:
Why was this changed? I'd think that we still want `wrap` as a defensive programming measure so that on extremely small screens, the content will wrap (even though this is not ideal in most case).

E.g. think of this website on an apple watch where the screen is super small.

We probably want the nav bar to go over 2 lines in this case